lib/idempo/active_record_backend.rb

```diff
@@ -2,7 +2,7 @@
 class Idempo::ActiveRecordBackend
   def self.create_table(via_migration)
     via_migration.create_table 'idempo_responses', charset: 'utf8mb4', collation: 'utf8mb4_unicode_ci' do |t|
-      t.string :idempotent_request_key, index: true, unique: true, null: false
+      t.string :idempotent_request_key, index: {unique: true}, null: false
```

Comment:
@julik This change should be mentioned in release notes. Users will have to manually re-create a UNIQUE index. (Not sure why, but this wrong syntax raised an exception in the tests, while it was simply ignored in production.)